Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionDataRecord.mm

```diff
@@ -196,22 +196,29 @@
         EXPECT_EQ(dataRecords.firstObject.totalSize, 237UL);
 
         [testController removeDataOfTypes:[NSSet setWithArray:@[ _WKWebExtensionDataTypeLocal, _WKWebExtensionDataTypeSession ]] forDataRecords:dataRecords completionHandler:^{
-            [testController fetchDataRecordsOfTypes:allDataTypesSet completionHandler:^(NSArray<_WKWebExtensionDataRecord *> *updatedRecords) {
-                EXPECT_EQ(updatedRecords.count, 1UL);
-
-                EXPECT_EQ(updatedRecords[0].errors.count, 0UL);
-
-                // Sync storage should still have data.
-                EXPECT_EQ(updatedRecords.firstObject.totalSize, 79UL);
-                removalComplete = true;
-            }];
+            removalComplete = true;
         }];
     }];
 
     TestWebKitAPI::Util::run(&removalComplete);
+
+    backgroundScript = Util::constructScript(@[
+        @"let usage = await browser.storage?.local?.getBytesInUse()",
+        @"browser.test.assertEq(usage, 0)",
+
+        @"usage = await browser.storage?.session?.getBytesInUse()",
+        @"browser.test.assertEq(usage, 0)",
+
+        @"usage = await browser.storage?.sync?.getBytesInUse()",
+        @"browser.test.assertEq(usage, 0)",
+
+        @"browser.test.notifyPass()"
+    ]);
+
+    Util::loadAndRunExtension(dataRecordTestManifest, @{ @"background.js": backgroundScript });
 }
 
-TEST(WKWebExtensionDataRecord, DISABLED_RemoveDataRecordsForMultipleContexts)
+TEST(WKWebExtensionDataRecord, RemoveDataRecordsForMultipleContexts)
```

Comment:
in the sysdiagnose for the bot that last ran these test, there's this error
`Failed to enable write-ahead logging on database for extension <private>: Error Domain=com.apple.WebKit.SQLite Code=5 "(null)"`

Code 5 is 
`define SQLITE_BUSY     5  /* The database file is locked */`

This can happen if another process is touching the file. maybe we're trying to delete this data while we're in the process of still adding it. adding a delay here to test that theory